ControlMeasurements/Startup.cs

```diff
@@ -33,6 +35,10 @@ public void ConfigureServices(IServiceCollection services)
 
 
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            var connection = @"Data Source =.; Initial Catalog = MeasureWater; Integrated Security = True";
```

Comment:
Zazwyczaj connection stringa bierzemy z konfiguracji (appsettings.json), ale na razie może być :)
Druga rzecz - widzę, że nazwałeś bazę "MeasureWater" - jeśli ta aplikacja ma tylko mierzyć liczniki z wody to spoko, jak inne typy to lepiej by było ją nazwać jakoś bardziej ogólnie.

Reply:
Chciałem dodac z appsetings.jason ale ten plik co mam w visual to tam nie mam connection stringa, jedynie jak utworze nowy taki sam to tam wyskakuje, tak ma byc?

co do nazwy to mea culpa. Poprawiłem pliki - wiec powinno byc lepiej :)

Reply:
Oryginalnie Ci się nie utworzył connection string zapewne dlatego, że nie miałeś podłączenia do bazy (a teraz już masz, więc pewnie Visual to ogarnia i dodaje odpowiednią sekcję). Tak jak pisałem, na razie to olej, w przyszłości przeniesiemy go do appsettingsów :)